apps/etterlatte-saksbehandling-ui/client/src/App.tsx

```diff
@@ -6,26 +6,43 @@ import { Person } from '~components/person/Person'
 import useInnloggetSaksbehandler from './shared/hooks/useInnloggetSaksbehandler'
 import nb from 'date-fns/locale/nb'
 import { registerLocale } from 'react-datepicker'
-import ErrorBoundary from '~ErrorBoundary'
+import ErrorBoundary, { ApiErrorAlert } from '~ErrorBoundary'
 import BrevOversikt from '~components/person/brev/BrevOversikt'
 import NyttBrev from '~components/person/brev/NyttBrev'
 import ScrollToTop from '~ScrollTop'
 import { ToggleNyOppgaveliste } from '~components/nyoppgavebenk/ToggleNyOppgavelist'
+import { isFailure, isSuccess, useApiCall } from '~shared/hooks/useApiCall'
+import { useEffect, useState } from 'react'
+import { ConfigContext, hentClientConfig } from '~clientConfig'
 
 function App() {
   const innloggetbrukerHentet = useInnloggetSaksbehandler()
   registerLocale('nb', nb)
 
+  const [fetchConfigResultStatus, hentConfig] = useApiCall(hentClientConfig)
+  const [config, setConfig] = useState({})
+
+  useEffect(() => {
+    hentConfig({}, (val) => setConfig(val))
```

Comment:
Ser ikke helt behovet for denne staten når `fetchConfigResultStatus` holder på samme data. Ville dog endret navn på den dersom du skal hente data fra den.